ci-operator/step-registry/openshift/e2e/gcp/libvirt/test/openshift-e2e-gcp-libvirt-test-commands.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+INSTANCE_PREFIX="${NAMESPACE}"-"${JOB_NAME_HASH}"
+
+mkdir -p "${HOME}"/.ssh
+mock-nss.sh
+chmod 0700 "${HOME}"/.ssh
+
+# gcloud compute will use this key rather than create a new one
+cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
+chmod 0600 "${HOME}"/.ssh/google_compute_engine
+cp "${CLUSTER_PROFILE_DIR}"/ssh-publickey "${HOME}"/.ssh/google_compute_engine.pub
+echo 'ServerAliveInterval 30' | tee -a "${HOME}"/.ssh/config
+echo 'ServerAliveCountMax 1200' | tee -a "${HOME}"/.ssh/config
+chmod 0600 "${HOME}"/.ssh/config
+
+# Copy pull secret to user home
+cp "${CLUSTER_PROFILE_DIR}"/pull-secret "${HOME}"/pull-secret
+
+gcloud auth activate-service-account --quiet --key-file "${CLUSTER_PROFILE_DIR}"/gce.json
+gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
+gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
+
+# TODO: ADD TESTS - HOW TO ADD openshift-tests binary? 
+cat  > "${HOME}"/run-tests.sh << 'EOF'
+#!/bin/bash
+set -euo pipefail
+export PATH=/home/packer:$PATH
+mkdir -p /tmp/artifacts/junit
+function run-tests() {
+  # Check the cluster operator output, status for available should be true for all operators
+  oc get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}' | grep -v True
```

Comment:
none should be progressing or degraded too, but I haven't checked how uch is this verified in regular e2e or it it just relies on the installer

Reply:
for now, I'm simply trying to get the test pod to run :)  I have to fix this up

Reply:
updated, added the same tests as are currently running w/ the template